transform-plugins/docs/Decoder-transform.md

```diff
@@ -12,5 +12,10 @@ Configuration
 **decode:** Specifies the configuration for decode fields; in JSON configuration, 
 this is specified as ``<field>:<decoder>[,<field>:<decoder>]*``.
 
-**schema:** Specifies the output schema; the fields that are decoded will have the same field 
+**schema:** Specifies the output schema; the fields that are decoded will have the same field
 name but they will be of type ``BYTES`` or ``STRING``.
+
+Conditions
+----------
+If error dataset is configured then all the erroneous rows, if present in the input, will be committed to the
+specified error dataset. If not configured, the erroneous rows will be committed to default error dataset.
```

Comment:
If nothing is specified, it actually doesn't write to any dataset and logs a warning in the logs. There is no "default" error dataset.